### PR TITLE
feat(TMST-621): Added PersonalStats component

### PR DESCRIPTION
**Pull request by lukakereceni**
[TMST-621](https://nidigitalsolutions.jira.com/browse/TMST-621)
[TMRX-1187](https://nidigitalsolutions.jira.com/browse/TMRX-1187)
[TMRX-1188](https://nidigitalsolutions.jira.com/browse/TMRX-1188)
[TMRX-1189](https://nidigitalsolutions.jira.com/browse/TMRX-1189)

**Description**
- Created PersonalStats component using the NK components
- Wrote unit tests for the PersonalStats component
- Wrote a story for the PersonalStats component
- Created mock data
- Implemented Puzzles theme (fonts, presets, etc.)
- Added tokens
- Updated Newskit to v7.2.0

https://user-images.githubusercontent.com/103412329/231980374-f666f5b5-d550-4298-8e9d-8fbda8f2a3bf.mov

[TMST-621]: https://nidigitalsolutions.jira.com/browse/TMST-621?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

[TMRX-1187]: https://nidigitalsolutions.jira.com/browse/TMRX-1187?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[TMRX-1188]: https://nidigitalsolutions.jira.com/browse/TMRX-1188?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[TMRX-1189]: https://nidigitalsolutions.jira.com/browse/TMRX-1189?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/ts-newskit/src/components/puzzles/personal-stats/index.tsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+type PersonalStatsItem = {
+  label: string;
+  value: number;
+};
+
+interface PersonalStatsProps {
+  data: PersonalStatsItem[];
+}
+
+const PersonalStatisticsContainer = styled(Block)<BlockProps>`
```
Comment (jasminaj0) [inline packages/ts-newskit/src/components/puzzles/personal-stats/index.tsx:21]: Hi @lukakereceni let's try to follow a pattern we have in other components directories, we can write styled components in styles.ts file and export them from there

File: packages/ts-newskit/src/components/puzzles/personal-stats/index.tsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+  Block,
+  TextBlock,
+  BlockProps,
+  TextBlockProps,
+  styled,
+  getMediaQueryFromTheme
+} from 'newskit';
+import WaveBg from '../../../assets/WaveBg';
+
+type PersonalStatsItem = {
```
Comment (jasminaj0) [inline packages/ts-newskit/src/components/puzzles/personal-stats/index.tsx:12]: Following the mentioned pattern, types should go in the types.ts file inside the component directory, in this case personal-stats

File: packages/ts-newskit/src/components/puzzles/personal-stats/mock/personalStats.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
```
Comment (jasminaj0) [inline packages/ts-newskit/src/components/puzzles/personal-stats/mock/personalStats.json:1]: Let's use fixtures folder instead mock for mock data

**Review by jasminaj0 — APPROVED** [review]

**Review by suneethanagella — APPROVED** [review]

File: packages/ts-newskit/src/components/puzzles/personal-stats/styles.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+    flex-direction: row;
+    gap: 40px;
+  }
+`;
+
+export const PersonalStatisticsItem = styled(Block)<BlockProps>`
+  display: flex;
+  align-items: center;
+  gap: 16px;
+`;
```
[the remark this answers is not in the document]
Comment (ibrahim-kurhan) [reply packages/ts-newskit/src/components/puzzles/personal-stats/styles.ts:40]: I think we need to use newskit tokens and stylePreset where possible

File: packages/ts-newskit/src/components/puzzles/personal-stats/personalStats.stories.mdx [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+import { PersonalStats } from './index.tsx';
+import { personalStatsItems } from './fixtures/personalStats.json'
+
+<Meta
+  title="Newskit Components/Puzzles/PersonalStats"
+  component={PersonalStats}
+/>
+
+# PersonalStats component
+
```
Comment (douglasmik) [inline packages/ts-newskit/src/components/puzzles/personal-stats/personalStats.stories.mdx:13]: component description please

**Review by ibrahim-kurhan — APPROVED** [review]

**Pull request merged**